database/migrations/2017_04_20_100848_create_monsters_table.php

```diff
@@ -60,7 +59,12 @@ public function up()
         });
 
         $prefix = DB::getTablePrefix();
-        DB::statement("ALTER TABLE {$prefix}monsters MODIFY base64_image MEDIUMBLOB");
+
+        $connection = config('database.default');
+
+        $columnType = config("database.connections.{$connection}.driver") === 'pgsql' ? 'BYTEA' : 'MEDIUMBLOB';
+
```

Comment:
```suggestion
        $columnType = DB::getDriverName() === 'pgsql' ? 'BYTEA' : 'MEDIUMBLOB';

```

Reply:
@pxpm can you use something like this here?

Reply:
What about SQLite? What about MongoDB? I think it'll fail for those. I say let's do a `switch` statement here, and for the databases we know what column type it should become, we increase the size. For those we do NOT know, we just leave it smaller (whatever `binary` does, so the equivalent of `BLOB`).

This would also help not do a DB statement at all for MongoDB for example - I imagine there's no `ALTER` statement there.